setup.py

```diff
@@ -1,21 +1,23 @@
 import setuptools
-import SwSpotify
 
 with open("README.md", "r") as fh:
     long_description = fh.read()
 
 setuptools.setup(
     name="SwSpotify",
-    version=SwSpotify.__version__,
```

Comment:
keep this as is, since then we wouldn't have to update the version number in 2 places before a release

Reply:
I changed setup.py and now it follow this answer https://stackoverflow.com/a/2073599/5774395 and https://github.com/psf/requests/blob/master/setup.py